src/Caliburn,Micro.Platform.Core.Tests/Caliburn,Micro.Platform.Core.Tests.csproj

```diff
@@ -4,8 +4,8 @@
     <TargetFramework>net6.0</TargetFramework>
     <RootNamespace>Caliburn_Micro.Platform.Core.Tests</RootNamespace>
     <Nullable>enable</Nullable>
-
     <IsPackable>false</IsPackable>
+    <LangVersion>10</LangVersion>
```

Comment:
Added this so it does not fail on unsupported Nullable setting.

Reply:
I think changing nullable to disable will fix the issue.

Reply:
It looks like the problem is that this test project (Caliburn,Micro.Platform.Core.Tests) targets net6.0 while there is a global.json requiring SDK 7.0.100. But what does this test project do anyway? It does contain no tests (has a spelling error in it's name) and does not exist on master. I guess it could simply be removed.

Reply:
it might not hurt to update the microsoft.tests.sdk to 17.5.0 also

Reply:
Looking at the test project that is failing.  I do not see any tests in it.  You could remove it until you add the tests you wanted

Reply:
Ok I'll remove that project tonight. Also won't add tests to it anyway cause I did not create this project in the first place 🤷‍♂️

Regarding updating test SDK. Is this the right moment or PR for that? My main intention was just getting the branch up to date with master before looking into the other Avalonia issues still left. But if you want to have the new version I can bump it of course.

IMO the projects could use some love in general. I noticed LangVersion 9, 10 and latest used + some other settings that are not in sync. They could all be extracted to the root props file.

Reply:
I can update the test sdk version in another pr.

Reply:
We could just close this PR and focus on this one: https://github.com/Caliburn-Micro/Caliburn.Micro/pull/846
It has the same changes + some more relevant things.